### DISCUSSION TITLE
error al ver la página de citas y los resultados de votaciones.

### DISCUSSION THREAD

**Discussion by galobart** [CarlosMiraGarcia/forocoches-premium#8] [General]
En la versión clásica.

**Comment by ghost** [comment]
Gracias por reportar el bug. Voy a echar un vistazo a ver si puedo reproducirlo.

**Comment by galobart** [comment]
Aparece la página en blanco y es en el navegador Brave, por si te sirve de algo.

 

De: ehtiotumolas ***@***.***> 
Enviado el: viernes, 9 de junio de 2023 21:13
Para: ehtiotumolas/forocoches-premium ***@***.***>
CC: galobart ***@***.***>; Author ***@***.***>
Asunto: Re: [ehtiotumolas/forocoches-premium] error al ver la página de citas y los resultados de votaciones. (Discussion #8)

 

Gracias por reportar el bug. Voy a echar un vistazo a ver si puedo reproducirlo.

**Reply by ghost** [reply]
He intentado recrear el bug, pero no lo consigo. 
Puedes hacer una screenshot si todavía te sigue dando problemas por favor?

**Comment by galobart** [comment]
Hola,

 

No sé si habrás publicado alguna nueva versión, pero ahora no me da error.  O igual son los de FC que habrán cambiado algo.

 

De: ehtiotumolas ***@***.***> 
Enviado el: jueves, 15 de junio de 2023 9:42
Para: ehtiotumolas/forocoches-premium ***@***.***>
CC: galobart ***@***.***>; Author ***@***.***>
Asunto: Re: [ehtiotumolas/forocoches-premium] error al ver la página de citas y los resultados de votaciones. (Discussion #8)

 

He intentado recrear el bug, pero no lo consigo.
Puedes hacer una screenshot si todavía te sigue dando problemas por favor?

**Reply by ghost** [reply]
Me alegro! Pues alguna versión publiqué no hace mucho. A veces Chrome/Brave tardan un poco en actualizar.
